src/lib/billing/types.ts

```diff
@@ -83,6 +83,7 @@ interface IUsageEvent {
 
 interface IBillableEvent extends IUsageEvent {
   readonly price: IPrice;
+  readonly quotaGUID: string;
```

Comment:
This might be better as a nullable type (I can't remember if the syntax is `string?` or `string | null`) because not all billable events will have quotas on them (presumably, since we only just started writing them).